Stripe/STPCard.m

```diff
@@ -135,6 +148,35 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+- (NSString *)description {
+    NSMutableArray *descriptionParts = [[NSMutableArray alloc] init];
+
+    // NSObject
+    [descriptionParts addObject:[NSString stringWithFormat:@"%@: %p", NSStringFromClass([self class]), self]];
```

Comment:
These lines are all very similar, is it possible to do this more automatically somehow to make it easier to add new properties? Eg. something like `propertyNamesToFormFieldNamesMapping`